M2/Macaulay2/packages/Normaliz.m2

```diff
@@ -261,7 +265,7 @@ doWriteNmzData(List):=(matrices)->
       4 => "inequalities",
       5 => "equations",
       6 => "congruences",
-      10 => "lattice_ideal",
+      10 => "normal_toric_ideal",
```

Comment:
Will this work with both versions of normaliz?

Reply:
See https://github.com/Macaulay2/M2/pull/2763.